src/settings-ui/Settings.UI/Controls/SettingsPageControl/SettingsPageControl.xaml

```diff
@@ -2,52 +2,55 @@
     x:Class="Microsoft.PowerToys.Settings.UI.Controls.SettingsPageControl"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Controls"
+    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
+    xmlns:converters="using:CommunityToolkit.WinUI.UI.Converters"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Microsoft.PowerToys.Settings.UI.Controls"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     Loaded="UserControl_Loaded"
-    xmlns:converters="using:CommunityToolkit.WinUI.UI.Converters"
-    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
     mc:Ignorable="d">
 
     <UserControl.Resources>
-        <converters:DoubleToVisibilityConverter x:Name="doubleToVisibilityConverter" GreaterThan="0" TrueValue="Visible" FalseValue="Collapsed" />
+        <converters:DoubleToVisibilityConverter
+            x:Name="doubleToVisibilityConverter"
+            FalseValue="Collapsed"
+            GreaterThan="0"
+            TrueValue="Visible" />
     </UserControl.Resources>
 
-    <Grid RowSpacing="24" Padding="20,0,0,0">
+    <Grid Padding="20,0,0,0" RowSpacing="24">
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
-            <RowDefinition Height="*"/>
- 
+            <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        
-        <TextBlock x:Name="Header"
-                   Text="{x:Bind ModuleTitle}"
-                   AutomationProperties.HeadingLevel="1"
-                   Style="{StaticResource TitleTextBlockStyle}"
-                   Margin="0,44,0,0"
-                   VerticalAlignment="Stretch"/>
+
+        <TextBlock
+            x:Name="Header"
+            Margin="0,44,0,0"
```

Comment:
This top margin is to align the text with the first NavItem, right? but is it _neccessary_?

Reply:
We should revisit this if we are going to introduce a custom title bar with Mica extending to the top. Leaving any changes out for now.